style.css

```diff
@@ -332,3 +336,89 @@ header a {
   margin: 0 24px;
   padding-left: 0;
 }
+
+/* contact section */
+.contact-section {
+  display: flex;
+  width: 100%;
+  justify-content: left;
+  background: white;
+  padding: 0;
+  flex-direction: column;
+  align-items: center;
+}
+
+html,
+body {
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+.contact-section h2 {
```

Comment:
- [ ] Kindly adhere to the design as I mentioned above which makes your design looks perfect and good to *eyes*. Actually building the design exactly the same as the figma `UI`, actually tell any person how much you are `detail-oriented` and indeed helps you to be the more professional developer who takes care of the `clients` requirements. Please have a look at the below image and one last thing make sure that your text looks exactly the same as per the requirement.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-10 17-04-08](https://github.com/ClarenceAtim/Portfolio/assets/66476329/2b7677a5-cd93-4c5e-aa24-2809be6976c7)

</td>
<td>

![Screenshot from 2023-05-10 17-04-19](https://github.com/ClarenceAtim/Portfolio/assets/66476329/bbb42eb4-37e0-44cd-83ce-e88b407b57cd)

</td>
</tr>
</table>


### Reminder

![image](https://github.com/geekelo/a-personal-portfolio-site/assets/66476329/c43ee2e3-782e-4080-875a-b7311226d8fa)